### PR TITLE
 #2980 added lodash and underscore detection

### PR DESCRIPTION
**Pull request by andreicristianpetcu**
Associated Issue: #2980

Here's the Pull Request Doc
https://devtools-html.github.io/debugger.html/CONTRIBUTING.html#pull-requests

### Summary of Changes
https://github.com/devtools-html/debugger.html/issues/2980#issuecomment-307108647

### Screenshots/Videos (OPTIONAL)
lo_dark
lo_light
underscore_dark
underscore_light

### CODE REVIEW

**Comment by jasonLaster** [comment]
@andreicristianpetcu just needs a quick rebase

**Comment by andreicristianpetcu** [comment]
Sorry for building on the wrong branch :D I rebased it.

**Comment by jasonLaster** [comment]
One small tweak

File: src/utils/frame.js [hunk 1]
```diff
@@ -47,6 +47,14 @@ function isExtJs(frame) {
...
 }
 
+function isUnderscore(frame) {
+  return getFrameUrl(frame).match(/underscore/i);
+}
+
+function isLodash(frame) {
+  return getFrameUrl(frame).match(/lodash/i);
+}
+
```
Comment (jdalton) [inline src/utils/frame.js]: For the regexp I'm not sure how loose you'd prefer but you might tighten it up to `/\blodash\b/`

File: src/utils/frame.js [hunk 2]
```diff
@@ -92,11 +92,11 @@ export function getLibraryFromUrl(frame: Frame) {
   }
 
   if (isUnderscore(frame)) {
-    return Object.assign({}, frame, { library: "Underscore" });
+    return "Underscore";
```
Comment (andreicristianpetcu) [inline src/utils/frame.js]: 👍 @jasonLaster

File: src/utils/frame.js (hunk 1 above)
Comment (jasonLaster) [inline src/utils/frame.js]: we're matching on the frame URL so there won't be any whitespace :)

Comment (jdalton) [inline src/utils/frame.js]: I was thinking more to avoid false positives for things like `lodashify`, `slodash`, or `underscoreize` kind of names.

**Pull request merged**